node/impl/full/eth.go

```diff
@@ -367,6 +368,29 @@ func (a *EthModule) EthGetTransactionCount(ctx context.Context, sender ethtypes.
 		return ethtypes.EthUint64(0), xerrors.Errorf("cannot parse block param: %s", blkParam)
 	}
 
+	// First, handle the case where the "sender" is an EVM actor.
```

Comment:
this comment should be on line 380?

Reply:
It's the entire section. I've put it into a block to make that clear.

Reply:
Eh, I switched it to an if chain.